configure.ac

```diff
@@ -222,9 +222,10 @@ AC_ARG_ENABLE([hashes],
         [--enable-hashes=HASHES],
         [Select hash algorithms to support.  Acceptable values are
          'all' or a comma-separated list of names from the file
-         'hashes.lst' and/or the keywords 'strong', 'freebsd',
-         'glibc', 'netbsd', 'openbsd', 'osx', and 'solaris',
-         which select groups of hashes as described in that file.
+         'hashes.lst' and/or the keywords 'strong', 'altlinux',
+         'freebsd', 'glibc', 'netbsd', 'openbsd', 'osx', 'owl',
+         'solaris', and 'suse' which select groups of hashes as
+         described in that file.
```

Comment:
Similarly, as long as we're touching this, please change "hash algorithms" to "hashing methods" and "groups of hashes" to "groups of methods".